mackerel-plugin-mongodb/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+prog=$(basename $0)
+if ! [[ -S /var/run/docker.sock ]]
+then
+	echo "$prog: there are no running docker" >&2
+	exit 2
+fi
+
+cd $(dirname $0)
+plugin=$(basename $(pwd))
+if ! which -s $plugin
```

Comment:
this requires target plugin (in this case `mackerel-plugin-mongodb`) to be installed within `$PATH`, but I'm happy if it works with `$0/mackerel-plugin-mongodb`

Reply:
Originally, I've wanted to run the plugin that was installed in somewhere. For example, if in development, the plugin might there in current directory. Or if in integration test, the plugin in might be installed in **/usr/bin** with the package manager.

So I'm going to add the plugin directory to front of the *PATH*. How do you feel about that?

Reply:
Looks good to me!